css/index.css

```diff
@@ -79,6 +79,46 @@ p {
     border-bottom: 1px dashed black;
 }
 
+.about-page .top-content {
+    display: block;
+    padding: 2% 2%;
+    margin-bottom: 30px;
+    border-bottom: 1px dashed black;
```

Comment:
This `border-bottom: 1px dashed black;` is overriding the same line in your original `.top-content` selector. You can omit this line.

Reply:
This is happening in a few other places. See if you can find them. :)